.github/ISSUE_TEMPLATE/accessibility-report.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Accessibility report
+about: Create an accessibility report
+---
+
+## Describe the accessibility issue
+
+_If this is a VMware product, do not fill this out! Instead consult with the accessibility team about the issue so we can consolidate all VMware related issues and track them separately._
+
+_Provide clear description of the accessibility issue and the impact to the user. Is this only in certain use cases? Is it confusing or broken?_
+
+## WCAG Guideline
+
+_What WCAG guideline is being violated here and how? You should reference a guideline from https://www.w3.org/TR/WCAG21/_
+
+## How to reproduce
+
+_Please provide a link to a reproduction scenario using one of the [Clarity Stackblitz templates](https://stackblitz.com/@clr-team). Reports without a clear reproduction may not be prioritized until one is provided._
+
+Steps to reproduce the behavior:
+
+1.  Go to '...'
+2.  Click on '....'
+3.  Scroll down to '....'
+4.  See error
+
+## Expected behavior
+
+_A clear and concise description of what you expected to happen._
+
+## Versions
+
+**App**
+
+- Angular: [e.g. 6]
+- Node: [e.g. 8.10.0]
+- Clarity: [e.g. 0.12.5]
+
+**Device:**
```

Comment:
Should we mention the 'officially' supported SR combos here and say that those issues take precedence/have higher priority?

Reply:
Done.